systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/ResourceManager.java

```diff
@@ -67,33 +68,43 @@ public static synchronized ResourceManager getInstance() {
         return pointerResources;
     }
 
-    public void setMethodResources() {
+    public synchronized void setMethodResources() {
```

Comment:
Does making these synchronized have any effect? I don't think tests would work if they were invoked concurrently anyway? And even if there were multiple invocations of this in parallel, wouldn't this cause undefined behavior if the pointers were set to different values?

Reply:
yeah good point, I will fix it